index.html

```diff
@@ -1,26 +1,36 @@
-<DOCTYPE html>
+<!DOCTYPE html>
 <html>
     <head>
-        <meta charset="utf-8">
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>Calculator</title>
         <link rel="stylesheet" href="css/style.css" type="text/css">
-        <script src="./js/script.js"></script>
-    </head>
+    </head> 
     <body>
         <div class="calculator">
-            <form class="calculator-form">
-                <div class="calculator-form__input">
-                    <input type="text" id="a" value="1">
-                    <input type="text" id="b" value="1"> 
-                </div>
-                <div class="calculator-form__action-bar action-bar"> 
-                   <input type="button" class="action-bar__button action-bar__button-sum" id="calc-sam" onclick="calcSum();" value="Sum">
-                   <input type="button" class="action-bar__button action-bar__button-sub" id="calc-sub" onclick="calcSub();" value="Sub">
-                   <input type="button" class="action-bar__button action-bar__button-mul" id="calc-mul" onclick="calcMul();" value="Mul">
-                   <input type="button" class="action-bar__button action-bar__button-div" id="calc-div" onclick="calcDiv();" value="Div">
-                </div>
-            </form>
-            <div class="calculator-result" id="result" ></div>
+            <div class="calculator-form action-bar">
+                <input type="text" class="calculator-form__item action-bar__output" id="calc-output"  disabled>
+                <input type="button" class="calculator-form__item action-bar__button-ce" id="CE" value="CE">
+                <input type="button" class="calculator-form__item number action-bar__button-seven" id="seven" value="7">
```

Comment:
I think it would be better to wrap digit buttons to a container (`.calculator__digit-panel`). It's up to you (depends on how you will re-structure BEM).

Reply:
I think it would be difficult to use grid if I wrap digit buttons to a container

Reply:
@NikitaOstrovsky636 it's not an argument. But for now, I'm OK with that markup.